workflows/src/main/resources/migration/account-info-schema.changelog-init.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="R3.Corda" id="create_account_info_table_id">
+        <createTable tableName="accounts">
+            <column name="identifier" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="host" type="NVARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="R3.Corda" id="accounts_pkey_id">
+        <addPrimaryKey columnNames="id" constraintName="accounts_pkey" tableName="accounts"/>
+    </changeSet>
+
+    <changeSet author="R3.Corda" id="accounts_id_idx_id">
+        <createIndex indexName="accounts_id_idx" tableName="accounts">
+            <column name="identifier"/>
+        </createIndex>
+    </changeSet>
+
+    <changeSet author="R3.Corda" id="accounts_host_idx_id">
+        <createIndex indexName="accounts_host_idx" tableName="accounts">
```

Comment:
I suspect your index name will need to match whatever it says in your JPA entity (I don't think it does currently). Same for the other indexes and the unique constraint.

Reply:
Done - thanks